Tests/SonarScanner.MSBuild.Shim.Test/PropertiesFileGeneratorTests.cs

```diff
@@ -1123,7 +1123,32 @@ public void ComputeProjectBaseDir_WorkingDirectory_FilesOutsideWorkingDirectory_
 
             sut.ComputeProjectBaseDir(projectPaths).FullName.Should().Be(@"C:\Solution");
             logger.AssertNoWarningsLogged();
-            logger.DebugMessages.Should().BeEquivalentTo(@"Using longest common projects path as a base directory: 'C:\Solution'.");
+            logger.DebugMessages.Should().BeEquivalentTo(
+                """
+                Using longest common projects path as a base directory: 'C:\Solution'. Identified project paths:
+                C:\Solution\Net\Name\Lib
+                C:\Solution\Net\Name\Src
+                C:\Solution\JS
+                """);
+        }
+
+        [TestMethod]
+        public void ComputeProjectBaseDir_BestCommonRoot_CaseSensitive()
+        {
+            var logger = new TestLogger();
+            var runtimeInformationWrapper = Substitute.For<IRuntimeInformationWrapper>();
+            runtimeInformationWrapper.IsOS(OSPlatform.Windows).Returns(false);
+            var additionalFileService = Substitute.For<IAdditionalFilesService>();
+            var sut = new PropertiesFileGenerator(new() { SonarOutputDir = @"C:\fallback"}, logger, new RoslynV1SarifFixer(logger), runtimeInformationWrapper, additionalFileService);
```

Comment:
I initially tried to write this test with Unit paths but since, on Windows, new DirectoryInfo("/").FullName is evaluated to "D:" (on my PC) I preferred to keep it simple and use windows full paths.

Reply:
I think that's okay, it makes sense.
What do you think of adding one more test with both `c:/` and `C:/` but with `IsOSPlatform` returning Windows?
So that we can see that they are treated the same?